test/unit/auth/test_jwt_auth.py

```diff
@@ -364,3 +364,99 @@ def test_refresh_instance_sends_post_request_with_correct_params(jwt_auth_init_a
     enterprise_id = 'fake_enterprise_id'
     with jwt_auth_init_and_auth_mocks(enterprise_id, 'enterprise', enterprise_id=enterprise_id) as oauth:
         oauth.refresh(None)
+
+
+@pytest.fixture()
+def jwt_subclass_that_just_stores_params():
+    class StoreParamJWTAuth(JWTAuth):
+        def __init__(self, **kwargs):
+            self.kwargs = kwargs
```

Comment:
Can you call `super()` here?

If you don't, then this test is incomplete. If we were to ever change our input params (e.g. say we rename a param), this test would continue to pass. Whereas if we call `super()`, then the super call would fail in such a scenario.

Since you are using `rsa_private_key_bytes`, that should work without any mocking needed.